Classes/Private/Manager/Video/OCTVideoEngine.m

```diff
@@ -90,6 +91,10 @@ - (BOOL)setupWithError:(NSError **)error
     AVCaptureConnection *conn = [self.dataOutput connectionWithMediaType:AVMediaTypeVideo];
     conn.videoOrientation = AVCaptureVideoOrientationPortrait;
```

Comment:
It would be better to call `orientationChanged` here instead of setting orientation, as it can be not Portrait by default.